Adds some honey foods | Changes mead recipe

TehZombehz opened the pull request:
Adds honey buns, honey nut bars (healthy!) and changes the recipe of mead to use honey instead of water and sugar (since we now have honey as a reagent).
![honeyfud3](https://cloud.githubusercontent.com/assets/15373249/14093950/7d588b96-f517-11e5-88fb-fb7ea4f2b23b.png)
:cl:
rscadd: Honey buns and honey nut bars are now craftable using honey.
tweak: Brewing mead now requires actual honey.
/:cl:

TehZombehz committed:
Adds some honey foods and changes mead

Adds honey buns, honey buns, and changes mead recipe to require honey.

TehZombehz committed:
Tiny graphic change

Rogue pixels bothered me

KorPhaeron commented on code/modules/food&drinks/recipes/drinks_recipes.dm:
```diff
@@ -364,7 +364,7 @@
 	id = "mead"
 	result = "mead"
-	required_reagents = list("sugar" = 1, "water" = 1)
+	required_reagents = list("honey" = 2)
```
How do I mix mead if its only honey?

TehZombehz replied:
You don't see it there because I didn't change it, but right below it is a thing saying 'CATALYST : ENYZME.'

TehZombehz replied:
I'm not THAT dumb, Kor.

KorPhaeron commented:
I think it might be a good idea to make the cooked food actually have the honey reagent in it, but that's up to you. I will merge once you reply to this comment with yes/no

TehZombehz committed:
Updates nutritional content with honey

Since honey supposedly also acts as a mild healing chemical, replacing
vitamin with honey seems optimal.

TehZombehz commented:
Good idea. Supposedly honey acts as a mild healing chemical, so making them contain honey both makes more sense and will make it healthier.
